easybuild/easyconfigs/o/orthAgogue/orthAgogue-20141105-foss-2023a.eb

```diff
@@ -0,0 +1,41 @@
+easyblock = 'CMakeMake'
+
+name = 'orthAgogue'
+version = '20141105'
+_commit = 'ebe3467'
+
+homepage = 'https://github.com/guyleonard/orthagogue'
+description = """
+orthAgogue: a tool for high speed estimation of homology relations within and between species
+in massive data sets. orthAgogue is easy to use and offers flexibility through a range of
+optional parameters.
+"""
+
+toolchain = {'name': 'foss', 'version': '2023a'}
```

Comment:
looks like this could be moved down to `gompi`?

Reply:
Yes, indeed. Thank you @migueldiascosta